l10n.toml

```diff
@@ -109,7 +109,3 @@ branches = [
 [[paths]]
   reference = "components/support/ktx/src/main/res/values/strings.xml"
   l10n = "components/support/ktx/src/main/res/values-{android_locale}/strings.xml"
-
-[[paths]]
-  reference = "components/feature/addons/src/main/res/values/strings.xml"
-  l10n = "components/feature/addons/src/main/res/values/values-{android_locale}/strings.xml"
```

Comment:
This removal doesn't seem intentional?

CC @Pike @Amejia481

Reply:
This is because https://github.com/mozilla-l10n/android-l10n/pull/127 hasn't been merged yet that picks up that change for android-l10n.

CC @tomprince , I guess the way to fix this would be to base the PR onto the last revision in https://github.com/mozilla-l10n/android-l10n/blob/master/_meta/mozilla-mobile-android-components.json ?

Reply:
Is landing the import PR blocked on something?

Reply:
Waits for a review by @Delphine. I flagged her explicitly.

Reply:
Done: https://github.com/mozilla-l10n/android-l10n/pull/127

Reply:
👍

Reply:
I think this pr should be re-based on the latest master of https://github.com/mozilla-l10n/android-l10n/.
Should we close this one and wait for the next time the bot try to update?

Reply:
The bot will update this PR, the next time it runs, as it uses the same branch.

Reply:
Got it thanks :)